tsconfig.json

```diff
@@ -6,13 +6,14 @@
     "declaration": true,
     "jsx": "react",
     "lib": [
-      "es2017",
+      "esnext",
```

Comment:
?

Reply:
need this to be able to do text.matchAll